lib/monitoring/aws-cloudwatch/LogMonitoring.ts

```diff
@@ -140,7 +164,21 @@ export class LogMonitoring extends Monitoring {
   }
 
   widgets(): IWidget[] {
+    const filterStatements: string[] = [];
+
     if (this.pattern) {
+      filterStatements.push(`filter @message like /${this.pattern}/`);
```

Comment:
If 50 is actually the limit, then we'd need to account for that if this is prepended too (i.e. 1 + 50).